Gemfile

```diff
@@ -1,6 +1,8 @@
 source 'https://rubygems.org'
-
```

Comment:
Not relevant to this diff, but you should go ahead and remove turbolinks. It's evil (not that I'm opinioinated or anything).